### PR TITLE
New command 'bzira issue bz 123'

### PR DESCRIPTION
**Pull request by pierreprinetti**
Add a Bugzilla link to an existing Jira card

### CODE REVIEW

File: bzira [hunk 1]
```diff
@@ -182,28 +217,12 @@ bz_tojira() {
...
-		}
-	}
-}
 EOF
 
 	issue_create < "$issue_payload" | tee "$issue_response_payload"
 
-	issue_id="$(jq -r '.key' "$issue_response_payload")"
-	issue_remotelink_create "$issue_id" < "$issue_remotelink_payload" > /dev/null
+	issue_key="$(jq -r '.key' "$issue_response_payload")"
```
Comment (adduarte) [inline bzira:224]: do you need the outside quotes?

Comment (pierreprinetti) [reply bzira:224]: In this case, I use quotes mostly out of habit.

1. I could remove the ‘-r’ flag and have ‘jq’ quote the value (it’s expected to be a string);
2. I could trust Jira not to use spaces or weird characters in keys.

That being said, putting quotes myself is a no-brainer.

Unless I am missing something?

**Comment by pierreprinetti** [comment]
@adduarte 
Is it good to merge?

**Review by adduarte — APPROVED** [review]
looks good

**Pull request merged**